javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/InstrumentationModule.java

```diff
@@ -50,6 +50,8 @@
   private static final Logger log = LoggerFactory.getLogger(InstrumentationModule.class);
   private static final Logger muzzleLog = LoggerFactory.getLogger("muzzleMatcher");
 
+  private static boolean classLoaderOptimizationOptOutFlag = Boolean.getBoolean("ClassLoaderOptimizationOptOut");
```

Comment:
it's traditional for Java system properties to start with lowercase
```suggestion
  private static boolean classLoaderOptimizationOptOutFlag = Boolean.getBoolean("classLoaderOptimizationOptOut");
```